src/main/proto_common/common.proto

```diff
@@ -258,6 +258,12 @@ message ProductDescription {
 
     // What kind of meeting type the product uses. Mandatory.
     MeetingType meetingType = 22;
+
+    // A list of lead customer fields this particular product requires.
+    repeated ContactField enforcedLeadPassengerFields = 23;
+
+    // A list of traveller this particular product requires.
```

Comment:
A list of traveler FIELDS (<-- missing) this prod....

Reply:
577482f has been pushed